src/streaming/TextTracks.js

```diff
@@ -442,16 +438,31 @@ function TextTracks() {
 
                 cue.onenter =  function () {
                     if (track.mode === 'showing') {
-                        log('Cue ' + this.startTime + '-' + this.endTime + ' : ' + this.cueHTMLElement.id + ' : ' + this.cueHTMLElement.innerText);
-                        captionContainer.appendChild(this.cueHTMLElement);
-                        scaleCue.call(self, this);
+                        var finalCue = document.createElement('div');
+                        log('Cue enter id:' + this.cueID);
+                        captionContainer.appendChild(finalCue);
+                        renderHTML(this.isd, finalCue,  function (uri) {
```

Comment:
It looks like v1.0.0 will have a different API for `renderHTML`. It no longer accepts a width and height parameters and instead reads `clientWidth` and `clientHeight` from `finalCue`.

This is an issue because `finalCue`'s `clientWidth` is 0, which causes all cues to have width 0. I worked around this by setting `finalCue.style.width = captionContainer.style.width;`, but I imagine there is a better way.